lib/ui/window.dart

```diff
@@ -515,8 +515,12 @@ class Locale {
 
 /// The most basic interface to the host operating system's user interface.
 ///
+/// It exposes the size of the display, the core scheduler API, the input event
+/// callback, the graphics drawing API, and other such core services.
+///
 /// There is a single Window instance in the system, which you can
-/// obtain from the [window] property.
+/// obtain from `WidgetsBinding.instance.window`, or `window` in `dart:ui` if
+/// `WidgetsBinding` is unavailable.
```

Comment:
I think we really want to incentivize the use of the singleton though, right? If `window` is imported directly from `dart:ui` then that usage cannot be dependency injected, can it? If I'm remembering that right then we should probably use some very strong language to push people towards the binding, if at all possible. What do you think?

Reply:
I see below that you elaborated on this. I would suggest including a little bit of the seriousness of the issue in this reference, too.

Reply:
I've tried to make it more serious and strong. Please let me know if it looks good, or if you have some sample text that I can follow (or copy/paste) :D